Team9Project/src/com/indragie/cmput301as1/ExpenseClaimListActivity.java

```diff
@@ -200,21 +239,74 @@ private void onManageTagsResult(Intent data) {
 				switch (mutation.getMutationType()) {
 				case DELETE:
 					claim.removeTag(tag);
+					if(filteredTagsList.contains(tag) && checkFilteredTags()) {
```

Comment:
It shouldn't be necessary to check if the list contains the tag before calling `remove()`, as calling `remove()` on a non-existent object doesn't do anything.